### PR TITLE
Add support for multisampled multiview to WebXR

### PR DESCRIPTION
**Pull request by cabanier**
Another attempt at landing support for multisampled multiview rendering for WebXR.
I tried to minimize code changes and let the developer opt in since not all content will work with multiview rendering.

*This contribution is funded by [Meta](https://meta.com)*

### CODE REVIEW

**Comment by cabanier** [comment]
@Mugen87 I'm getting an error in `webgpu_postprocessing_afterimage` but I didn't make changes to that code. The file also seems to render ok.

**Comment by Mugen87** [comment]
We have some issues with E2E testing in context of WebGPU examples, see https://github.com/mrdoob/three.js/pull/27447#issuecomment-1870143929. The failing test in this PR (and others) will not prevent a merge^^.

**Comment by cabanier** [comment]
> We have some issues with E2E testing in context of WebGPU examples, see [#27447 (comment)](https://github.com/mrdoob/three.js/pull/27447#issuecomment-1870143929). The failing test in this PR (and others) will not prevent a merge^^.

Good to know! I will ignore it then :-)
Any comments about this PR? I know there were several attempts to land this and I'm unsure what stopped it previously.

**Comment by cabanier** [comment]
@Mugen87 are there fundamental problems with this PR or do you think multiview is not worth the effort?

**Comment by Mugen87** [comment]
To me there is no fundamental issue with this PR. I think `GL_OVR_multiview` (or any sort of multiview technique) is a must for efficient XR rendering and the engine should eventually support it. 

I like the idea of having a parameter that enable a separate code path for `GL_OVR_multiview`. Maybe I would rename it to just `multiview` though so it better matches `alpha` and `antialias`.

Could this parameter also be part of the WebXR interface? Meaning:
```js
renderer.xr.enabled = true;
renderer.xr.multiview = true;
```
If possible, I would prefer this solution.

However, it is up to @mrdoob to make the final decision for this change and how the integration should look like. The previous implementation was removed via #18750 with the comment:

> This feature was added prematurely and it's currently unmaintained. We can reconsider adding it again when browsers add support in the future.

I would say this PR is a good opportunity to make this reconsideration.

**Comment by cabanier** [comment]
> I like the idea of having a parameter that enable a separate code path for `GL_OVR_multiview`. Maybe I would rename it to just `multiview` though so it better matches `alpha` and `antialias`.

done

> Could this parameter also be part of the WebXR interface? Meaning:
> 
> ```js
> renderer.xr.enabled = true;
> renderer.xr.multiview = true;
> ```
> 
> Or is it a problem when `multiview` is toggled?

I felt that this was better store in the webglrenderer just like we do for multisampled_render_to_texture since it's a different render mode. I initially added the flag to the webxrmanager but that resulted in messier code on the rendering side.
Multiview could be toggled between vr sessions but I don't know why a site would do that. Also, a site might want to create another render target to render to a layer and it might not need multiview (ie if it renders to a mono layer)

File: examples/jsm/controls/OrbitControls.js [hunk 1]
```diff
@@ -1418,7 +1448,8 @@ class OrbitControls extends EventDispatcher {
 		scope.domElement.addEventListener( 'contextmenu', onContextMenu );
 
 		scope.domElement.addEventListener( 'pointerdown', onPointerDown );
-		scope.domElement.addEventListener( 'pointercancel', onPointerUp );
+		scope.domElement.addEventListener( 'pointercancel', onPointerCancel );
+		scope.domElement.addEventListener( 'pointerout', onPointerOut );
```
Comment (Mugen87) [inline examples/jsm/controls/OrbitControls.js:1452]: Are the changes to `OrbitControls` related to #27333?

Ideally, these are reverted before the PR is merged 😅 .

Comment (cabanier) [reply examples/jsm/controls/OrbitControls.js]: oops :-)
I removed those erroneous changes from the PR

**Comment by cabanier** [comment]
@mrdoob ping?

**Comment by cabanier** [comment]
@mrdoob ping?

**Pull request closed without merging**